tests/jenkins/install.sh

```diff
@@ -23,8 +23,8 @@ cd $WORKSPACE/build_${CORENRN_TYPE}
 echo "${CORENRN_TYPE} build"
 if [ "${CORENRN_TYPE}" = "GPU" ]; then
     cmake \
-        -DCMAKE_C_FLAGS:STRING="-O2 -ta=tesla:cuda9.0" \
-        -DCMAKE_CXX_FLAGS:STRING="-O2 -ta=tesla:cuda9.0" \
+        -DCMAKE_C_FLAGS:STRING="-O2 -ta=tesla:cuda9.2" \
```

Comment:
could you check by dropping `-ta=tesla:cuda9.2` part here?

Reply:
It runs without it as well. Shall I remove this part?